api/src/test/resources/registration_app.json

```diff
@@ -140,35 +140,6 @@
               ]
             }
           ]
-        },
-        {
-          "id": "ethnicity",
-          "label": "Ethnicity",
-          "questions": [
-            {
-              "legend": "acme.app.registerPatient.section.ethnicity",
-              "fields": [
-                {
-                  "type": "obs",
-                  "formFieldName": "obs.ACME:912",
-                  "label": "acme.app.registerPatient.ethnicity",
-                  "widget": {
-                    "providerName": "acmeforms",
-                    "fragmentId": "field/ethnicity"
```

Comment:
Replace this with `"fragmentId": "field/text"`.

Reply:
Replaces as well.  Thank you very much @mks-d. Just learnt a tone. :-)